charts/artifact-registry/chart/scripts/install_helm.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+apt-get update && apt-get upgrade
+apt-get install curl gnupg2 --yes
+curl https://baltocdn.com/helm/signing.asc >> signing.asc
+cat signing.asc | apt-key add -
+apt-get install apt-transport-https --yes
+echo "deb https://baltocdn.com/helm/stable/debian/ all main" | tee /etc/apt/sources.list.d/helm-stable-debian.list
```

Comment:
We don't need `tee`. `tee` is often used together with `sudo` to write to a file as non-root, but we aren't using `sudo` as we are already root.
```suggestion
echo "deb https://baltocdn.com/helm/stable/debian/ all main" > /etc/apt/sources.list.d/helm-stable-debian.list
```